.github/workflows/go.yml

```diff
@@ -25,9 +25,47 @@ jobs:
       working-directory: ./api-gateway
       run: |
           ls -la
+    
+    - name: Build
```

Comment:
this workflow runs only ls -la on the current directory that you're in not building any go binaries. pay attention to the run part!!